components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -1169,6 +1180,7 @@ private void removeAuthenticationResult(OAuthMessage oAuthMessage, String sessio
         }
     }
 
+    @Deprecated
```

Comment:
For **private** methods we can directly change the method signature. We do not need to deprecate and introduce overloaded methods